pages/bare_metal_cloud/dedicated_servers/esxi-partitioning/guide.fr-fr.md

```diff
@@ -0,0 +1,172 @@
+
+**Cet article a pour objectif de vous montrer comment choisir un schéma de partitionnement dans l'[espace client OVHcloud](https://www.ovh.com/manager/#/dedicated/configuration) ou l’[API OVHcloud](https://api.ovh.com/).**
+
+> [!primary]
+>
+> ESXi 7.0 est souvent utilisé comme exemple sur cette page, mais la documentation est valable pour les versions 7.0 et ultérieures d'ESXi.
+>
+
+## Prérequis
+
+- Un [serveur dédié](https://www.ovhcloud.com/fr/bare-metal/) dans votre compte OVHcloud, **prêt à être installé/réinstallé**.
+- Avoir accès à l'[API OVHcloud](https://api.ovh.com/).
+
+> [!alert]
+>
+> La réinstallation d'un serveur dédié supprime toutes les données qui y sont actuellement stockées.
+>
+
+## En pratique
+
+ESXi 7.0 introduit une [option d'amorçage permettant de configurer la taille de la partition système ESXi](https://kb.vmware.com/s/article/81166). Cette fonctionnalité a été introduite par l'éditeur parce que certains clients se plaignaient de la présence parfois superflue d'un datastore qui remplissait tout l'espace disque ou un d'un OS qui avait une partition système surdimensionnée. OVHcloud propose désormais cette fonctionnalité qui est disponible aussi bien depuis l'[espace client OVHcloud](https://www.ovh.com/manager/#/dedicated/configuration) que via l’[API OVHcloud](https://api.ovh.com/).
+
+Malgré le fait que votre serveur ait plusieurs disques, l'installation d'ESXi n'est possible que sur le premier disque de la grappe de disques sélectionnée pour l'installation (voir « [Choisir une grappe de disques pour installer un système d’exploitation](/pages/bare_metal_cloud/dedicated_servers/install_hybrid) » ), les autres disques pouvant être configurés par la suite par l'utilisateur pour être utilisés pour stocker les machines virtuelles (voir « [Configuration du stockage d'un serveur HGR-STOR-2](/pages/bare_metal_cloud/hgrstor2_system_configuration#add-datastore) » ).
+
+4 valeurs sont possibles :
+
+|Valeur|Taille Système¹|Datastore³|
+|---|---|---|
+|`default`|130 Gio|Tout l'epace restant²|
+|`min`|32 Gio|Tout l'epace restant²|
+|`small`|64 Gio|❌⁴|
+|`max`|Tout l'espace disponible²|❌⁴|
+
+¹ Seulement le premier disque de la grappe de disques sélectionnée pour l'installation de l'OS.<br />
+² Espace disque sur lequel l'OS va être installé.<br />
+³ Un datastore est une partition de disque (parfois aussi appelée « container ») que ESXi va utiliser pour stocker ses machines virtuelles. Retrouvez plus de détails sur [cette documentation VMware (EN)](https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.storage.doc/GUID-5EE84941-366D-4D37-8B7B-767D08928888.html).<br />
+⁴ Le client pourra toujours [ajouter un datastore](/pages/bare_metal_cloud/hgrstor2_system_configuration#add-datastore) par la suite, mais uniquement sur les autres disques.<br />
+
+Comme vous pouvez le constater, tout l'espace du disque d'installation est utilisé sauf pour le schéma `small`.
+
+> [!primary]
+>
+> La saviez-vous ?
```

Comment:
```suggestion
> Le saviez-vous ?
```
And as I said on the en version, I'm not sure we want to commit to this always being true.